orc/src/main/java/org/apache/iceberg/orc/ORCSchemaUtil.java

```diff
@@ -23,7 +23,9 @@
 import java.util.Objects;
 import java.util.Optional;
 import java.util.stream.Collectors;
+import org.apache.hadoop.conf.Configuration;
```

Comment:
Wonder whether the "whole `Configuration` spiel" is necessary here. Hadoop dependencies are already `compileOnly` to eventually get rid of those entirely. Wouldn't a simple boolean flag do the same thing in this class?